x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/exceptions/alerts_table_flow/rule_exceptions/closing_all_matching_alerts.cy.ts

```diff
@@ -77,4 +101,54 @@ describe('Close matching Alerts ', { tags: ['@ess', '@serverless'] }, () => {
     cy.get(LOADING_INDICATOR).should('not.exist');
     cy.get(ALERTS_COUNT).should('contain', '1');
   });
+
+  it('Should close all alerts from if several rules has shared exception list', () => {
+    cy.get(ALERTS_COUNT).should('contain', '3');
+
+    let exceptionListId = '';
+    createExceptionList(getExceptionList1(), getExceptionList1().list_id)
+      .then((response) => {
+        exceptionListId = response.body.id;
+        createExceptionListItem(getExceptionList1().list_id, {
+          item_id: '123',
+          entries: [
+            {
+              field: 'user.name',
+              operator: 'included',
+              type: 'match_any',
+              value: ['alice'],
+            },
+          ],
+        });
+      })
+      .then((response) => {
+        return createRule(
+          getNewRule({
+            exceptions_list: [
+              {
+                id: response.body.id,
+                list_id: getExceptionList1().list_id,
+                type: getExceptionList1().type,
+                namespace_type: getExceptionList1().namespace_type,
+              },
+            ],
+          })
+        );
+      })
+      .then(() =>
+        linkRulesToExceptionList('rule_testing', {
+          id: exceptionListId,
+          listId: getExceptionList1().list_id,
+        })
+      );
+
+    goToExceptionsTab();
+    cy.reload();
+    openEditException();
+    selectBulkCloseAlerts();
+    submitEditedExceptionItem();
+    goToAlertsTab();
+
+    cy.get(ALERTS_COUNT).should('not.exist');
```

Comment:
Is there a better assertion we can use here? Asserting the presence of "No alerts" or "0 alerts" or something like that feels much less prone to false positives than this approach.

Reply:
Added: `cy.get(ALERT_EMBEDDABLE_EMPTY_PROMPT).should('exist');`